### PR TITLE
Create example of how to render smiles and import

### PR DESCRIPTION
**Pull request by aislinblack**
(will be integrated into popper later)

I will explain in a comment how to test this.

### CODE REVIEW

File: app/static/js/cluster.js [hunk 1]
```diff
@@ -102,6 +102,17 @@ fetch('js/data.json', {mode: 'no-cors'})
...
+  /*
+  var mol = data.nodes[0].data.id;
+  let smilesDrawer = new SmilesDrawer.Drawer(options);
+  SmilesDrawer.parse(mol, function(tree) {
+    // Draw to the canvas
+    smilesDrawer.draw(tree, "drawing", "light", false);
+    // Alternatively, draw to SVG:
+    // svgDrawer.draw(tree, 'output-svg', 'dark', false);
+  });
+  */
```
Comment (aislinblack) [inline app/static/js/cluster.js:114]: Uncomment this section and then run to test that the mol draws

**Comment by aislinblack** [comment]
[Draw Chemical Structure](https://trello.com/c/2khXw6fz/9-draw-chemical-structure)

**Comment by aislinblack** [comment]
http://doc.gdb.tools/smilesDrawer/
This is the library I have used

File: app/static/js/cluster.js (hunk 1 above)
Comment (harbourcd) [reply app/static/js/cluster.js:114]: Looks good! should we uncomment before merging?

Comment (aislinblack) [reply app/static/js/cluster.js:114]: Nope! I'm leaving it like this until @klu17 gets her popper pr merged that way we can get the drawing integrated into those

**Review by harbourcd — APPROVED** [review]
Looks good!

**Pull request merged**